src/ServiceManagement/Compute/Commands.ServiceManagement/Generated/Deployment/InvokeAzureDeploymentBeginDeletingRoleInstanceByBeploymentSlotMethod.cs

```diff
@@ -0,0 +1,49 @@
+// 
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// 
+
+// Warning: This code was generated by a tool.
+// 
+// Changes to this file may cause incorrect behavior and will be lost if the
+// code is regenerated.
+
+using System.Management.Automation;
+using Microsoft.WindowsAzure.Management.Compute;
+
+namespace Microsoft.WindowsAzure.Commands.Compute.Automation
+{
+    [Cmdlet("Invoke", "AzureDeploymentBeginDeletingRoleInstanceByBeploymentSlotMethod")]
+    public class InvokeAzureDeploymentBeginDeletingRoleInstanceByBeploymentSlotMethod : ComputeAutomationBaseCmdlet
```

Comment:
For this first version, I would concentrate on the synschronous methods.  Later on, we can put in some default OM around polling in PowerShell based on the return type of the method (Basically a Begin method and an end method, or a begin method, a check status, and an end).  If we had extra information, we could generate this by default.